### PR TITLE
ProgressBar CSS improvements

### PR DESCRIPTION
**Pull request by dmandrioli**
- Add a LESS variable @height for consistency with @line-height. These two variables control the default height of the widget and the default vertical position of the text
 - Ensure inner font size inherit the widget font-size
 - Add some comments, remove a "overflow:hidden" not needed

### CODE REVIEW

File: ProgressBar/themes/bootstrap/variables.less [hunk 1]
```diff
@@ -1,2 +1,3 @@
-@transition-params: 0.3s linear 0s;
-@line-height: 1.6em;
\ No newline at end of file
+@height: 1.8em; // Controls the height of the widget (180% of the font-size)
```
Comment (cjolif) [inline ProgressBar/themes/bootstrap/variables.less]: isn't that "dangerous" to have such non-scoped LESS (i.e. not pb-height for example) variable naming just for a particular widget? What is our practice elsewhere? Can this collapse with other widget defining a similar variable?

Comment (dmandrioli) [inline ProgressBar/themes/bootstrap/variables.less]: Not sure if it's dangerous or not, probably not, but there is definitely something to do on LESS consistency across widgets.
There are variables such as @star-icon-size, others like @d-switch-w or too short names like @height.

Where are defined these LESS variables is also not consistent:
- In a variables.less for the Switch in `Switch/variables.less` which is common to all themes.
- in the theme LESS file for others

About naming, I propose to follow the pattern of @d-switch-xxx for all widgets.
About where to define LESS variables, having a variables.less for each widget could be interesting in the perspective of a theming tool, to facilitate a kind of automatic introspection mechanism of all LESS variables. But I guess it's not the time to do that and we should remove the only variables.less of Switch.

I would prefer to refactor all LESS variables in a unique commit after merging all CSS related PRs.

Comment (cjolif) [inline ProgressBar/themes/bootstrap/variables.less]: > I would prefer to refactor all LESS variables in a unique commit after merging all CSS related PRs.

Ok. Can you create an issue for that? (yes one more...)

Comment (dmandrioli) [inline ProgressBar/themes/bootstrap/variables.less]: > Ok. Can you create an issue for that? (yes one more...)
Sure, https://github.com/ibm-js/deliteful/issues/496.

Comment (AdrianVasiliu) [inline ProgressBar/themes/bootstrap/variables.less]: > Can this collapse with other widget defining a similar variable?

On a related note, shouldn't we systematically apply a naming pattern (with the "d-" prefix) also for CSS classes? I'm not sure whether we have other cases, the only one I have in mind is the `fill` class (https://github.com/ibm-js/deliteful/blob/0.6.0-alpha/LinearLayout/themes/LinearLayout_template.less#L12). I can create an issue for it.

Comment (dmandrioli) [inline ProgressBar/themes/bootstrap/variables.less]: We also have https://github.com/ibm-js/deliteful/blob/master/docs/Button.md#styling-css-classes
Feel free to create an issue.

**Comment by dmandrioli** [comment]
Closed by d984dc8914b7844659a81ebc80ca45dfaa984453

**Pull request closed without merging**